third_party/kmod/CMakeLists.txt

```
@@ -0,0 +1,32 @@
+# build the kernel module
+set(MODULE_NAME kernel_data)
+set(MODULE_SOURCE_DIR ${PROJECT_SOURCE_DIR}/third_party/kmod)
+set(MODULE_SOURCE_FILES ${PROJECT_SOURCE_DIR}/third_party/kmod/${MODULE_NAME}.c)
+set(MODULE_FILE ${MODULE_NAME}.ko)
+
+# CMAKE_SYSTEM_VERSION defaults to CMAKE_HOST_SYSTEM_VERSION, which is
+# documented to use `uname -r` on systems (like Linux) that support `uname`.
+# https://cmake.org/cmake/help/latest/variable/CMAKE_HOST_SYSTEM_VERSION.html
+set(KERNEL_SRC "/lib/modules/${CMAKE_HOST_SYSTEM_VERSION}/build" )
```

```suggestion
set(KERNEL_SRC "/lib/modules/${CMAKE_SYSTEM_VERSION}/build" )
```

I think `CMAKE_SYSTEM_VERSION` is slightly more correct here than `CMAKE_HOST_SYSTEM_VERSION` since we're looking for the version we're compiling *for* -- we might have headers for multiple kernels installed, for example.

This will almost certainly never make a difference, but we either have to rewrite this variable or the comment anyway and I'd prefer we change the variable.